example/spec/serial/AsyncOperationRunnerSuccessfulLambdaSpec.js

```diff
@@ -108,15 +108,12 @@ describe('The AsyncOperation task runner executing a successful lambda function'
   it('returns the updated record from GET /asyncOperations', async () => {
     if (beforeAllError) fail(beforeAllError);
     else {
-      const response = await listAsyncOperations({
```

Comment:
@jennyhliu  you should take at least a quick look at this to make sure it doesn't conflict/cause issues with your LIST PR.

Reply:
If you make this change, then listAsyncOperations is not integration-tested. I suggest to keep the old implementation.

Reply:
@jennyhliu  I agree that's a change, however integration testing list as a side effect is not necessarily for the purpose of this test,  and not entirely consistent with our API testing generally - I don't believe we integration test most of our API query endpoints (e.g. granules POST), *should* this be different? 

If we are intending on testing that the endpoint works in integration, should we not make an integration test targeting that behavior instead of as a side effect of verifying the lambda ran and reported successful?

Reply:
Actually we do call createGranule a few places, and there are integration tests to cover api (not throughly though): example/spec/parallel/testAPI/granuleSpec.js

Reply:
Right - but that's a specific API test.  I'm happy to add a targeted test to this ticket, if we feel integration testing the API is needed for search, although I still think it's a fair question what >integration< other than database connection to the API we're actually testing by doing so, over and above the units.

Reply:
Ok.

Reply:
To demonstrate/highlight/give a target for discussion, see: https://github.com/nasa/cumulus/pull/3775    I don't think this adds much to what the units already do and adds more risk to side-effects on other tests, but am open to discussion on this.   Let me know what you think.

Reply:
I'm fine with letting unit test cover listAsyncOperations